pkgs/development/python-modules/diskcache/default.nix

```diff
@@ -28,8 +28,6 @@ buildPythonPackage rec {
     mock
   ];
 
-  disabled = lib.versionAtLeast django.version "2.0";
-
   checkPhase = ''
     pytest
```

Comment:
Please switch to pytestCheckHook.

Reply:
done (I think). Tests pass + dependencies still work.